Added a new paragraph in the contribution guide about an error because it occurred to me and may occur to others also. (#11131)
grafana/loki: CONTRIBUTING.md

3. Run the command `make docs`. This uses the `grafana/docs` image which internally uses Hugo to generate the static site.
4. Open http://localhost:3002/docs/loki/latest/ to review your changes.

**Remember:** If running `make docs` command gave you the following error.

   - `path /tmp/make-docs.Dcq is not shared from the host and is not known to Docker.`
   - `You can configure shared paths from Docker -> Preferences... -> Resources -> File Sharing.`

> Then you can go to Docker Desktop settings and open the resources, add the path `/tmp` or the specific path `/tmp/make-docs.Dcq` and click **Apply** for the changes to take effect.

Reviewer: I might be tempted to only document adding the parent temporary directory (or the value of TMPDIR if it is set) because the script creates different subdirectories each time.

```suggestion
Then you can go to Docker Desktop settings and open the resources, add the temporary directory path `/tmp`.
```
